packages/shared/locales/en.json

```diff
@@ -759,6 +759,12 @@
         "confirmDeveloperProfile": {
             "title": "You are creating a developer profile",
             "body": "You will be connected to the Chrysalis Devnet by default, and can switch networks manually. Only continue if you know what you are doing."
+        },
+        "privacyPolicyUpdate": {
+            "title": "Privacy Policy",
+            "body": "We have made some modifications to the way we share and disclose your personal data when you use Firefly.",
+            "checkbox": "I've read and I accept the updated Privacy Policy",
```

Comment:
This phrasing could possibly be a bit smoother, perhaps `I have read and accept the updated Privacy Policy`?

Reply:
Yeah I agree. I copied this wording from here to keep it consistent, but we can probably change it:

https://github.com/iotaledger/firefly/blob/3669df71cab115f949ba18e4dc320bf12cdbb030/packages/shared/locales/en.json#L10

Reply:
Ah didn't notice that it was there as well. Hmm, is probably worth adjusting.